src/App/Controllers/Modules/GetContent.php

```diff
@@ -94,11 +94,24 @@ protected function getData($dir)
      */
     protected function getFolderContent($folder, $rec = false)
     {
-        return $this->storageDisk->listWith(
-            ['mimetype', 'visibility', 'timestamp', 'size'],
-            $folder,
-            $rec
-        );
+        return $this->storageDisk->listContents($folder, $rec)->map(function ($attributes) {
+            $path = $attributes->path();
+            $name = substr($path, strrpos($path, '/'));
+            $extension = substr($name, strrpos($name, '.') + 1);
+            return [
+                'path' => $path,
+                'dirname' => str_replace("/$name", '', $path),
+                'basename' => $name,
+                'extension' => $extension,
+                'filename' => substr($name, 0, strrpos($name, '.')),
+                'timestamp' => $attributes->lastModified(),
+                'size' => $attributes->isFile() ? $attributes->fileSize() : 0,
+                'type' => $attributes->isFile() ? 'file' : 'dir',
+                'mimetype' => $attributes->isFile() ? ($this->storageDisk->mimeType($path)) : 'folder',
+                'etag' => $attributes->extraMetadata()['ETag'] ?? null,
```

Comment:
do we need this ?

Reply:
With this we format the information per folder/file in the FileManager.

Reply:
i meant the etag, i cant recall that i used it b4, maybe i forgot but if there's no use for it then we can remove it.